Makefile

```diff
@@ -120,6 +130,26 @@ DOCKER_ARCH_TAG=$(ZARCH)
 
 FULL_VERSION:=$(ROOTFS_VERSION)-$(HV)-$(ZARCH)
 
+
+ifeq ($(PREEMPT_RT),1)
+  ifneq ($(ZARCH), amd64)
+    $(error "PREEMPT_RT is not supported for $(ZARCH)")
+  endif
+    KERNEL_FLAVOR=rt
+endif
+
+KERNEL_VERSION=$(KERNEL_VERSION_$(KERNEL_FLAVOR))
+KERNEL_COMMIT=$(KERNEL_COMMIT_$(ZARCH)_$(KERNEL_FLAVOR))
+
+# this is the branch we use for kernel builds. Can be overwritten from a command line
+KERNEL_BRANCH ?= eve-kernel-$(ZARCH)-$(KERNEL_VERSION)-$(KERNEL_FLAVOR)
+# this can be set to 'latest' from the command line to use development kernel
+KERNEL_DOCKER_TAG ?= $(KERNEL_BRANCH)-$(KERNEL_COMMIT)-$(KERNEL_COMPILER)
```

Comment:
The KERNEL_BRANCH starts with eve-kernel, which means that the tag would do the same. But the tag is presumably appended as lfedge/eve-kernel:<tag>, meaning that eve-kernel would be repeated. That doesn't make sense.

Reply:
the resulting image name looks like this `docker.io/lfedge/eve-kernel/eve-kernel-amd64-v6.1.38-generic-29700903aee7-gcc` tag repeats branch name because all docker images are stored in the same repo docker.io/lfedge/eve-kernel. For easier troubleshooting it is nice to see full branch name in `images/rootfs.yml`

Reply:
OK